core/src/components.d.ts

```diff
@@ -750,6 +750,10 @@ export namespace Components {
           * Display an outline style button.
          */
         "outline": boolean;
+        /**
```

Comment:
Based on previous comments on Card task, the default shape of Shape can't be an option. So, to keep consistency, we can remove the round shape as option. A task was already created to investigate,  [FW-6118](https://outsystemsrd.atlassian.net/browse/FW-6118) .

I agree with @BenOsodrac related to have all the options available and supported on the component context.

Reply:
@brandyscarney should I revert this then?

Reply:
Yeah, let's remove the default shape for now. I added it to [FW-6118](https://outsystemsrd.atlassian.net/browse/FW-6118) at Bruno's suggestion so we can track adding it there.

Reply:
Are we defaulting to `"soft"` for now?

Reply:
Yeah, I double-check with UX & PO and the default should always be `soft`. I had it wrong in the first place.

Already committed the changes about `soft` removal!

Reply:
Oh okay, thank you! I updated that ticket to reflect `soft` as the default now. 👍